training/vision/components/image_classification/spec.yaml

```diff
@@ -4,8 +4,8 @@ type: command
 description: Trains an image classification model. [Learn More](https://aka.ms/built-in-vision-components)
 
 name: train_image_classification_model
-display_name: Train Image Classification Model (Private Preview)
-version: 0.0.3
+display_name: Image Classification AutoML
```

Comment:
"Image Classification AutoML legacy models"?

I am trying to find a way to formulate that doesn't imply that we are doing automl

Reply:
display_name: Image Classification AutoML Legacy Model Finetune
description: Component to finetune AutoML legacy models for image classification.

Reply:
updated.